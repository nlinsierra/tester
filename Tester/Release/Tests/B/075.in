3C 2H 4H 4C 6C TH 9H 9C TC 6S 3D 5D 9S AC TD 7C AD 5S 7S 8C 3S 2C JD 7H 7D QS 6H 2S 6D 4S KD TS JH 8D 9D AS 3H 2D JC 4D KC KS QD JS
