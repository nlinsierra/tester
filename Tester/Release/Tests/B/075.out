8
5C QC 5H 8H QH KH AH 8S
